builder-model/.gitignore

```diff
@@ -0,0 +1,11 @@
+build
```

Comment:
I don't think this project should load as it's own project.  It's part of the greater gradle build.  Anyway, the parent .gitignore should cover all this.